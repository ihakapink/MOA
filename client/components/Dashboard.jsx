import React from 'react'
import Header from './Header'

export default class Dashboard extends React.Component{
    render(){
        return (
            <React.Fragment>
            <Header />
            <p>YESSSSSS SIS this is working  from the dashboard</p>
            </React.Fragment>
        )

        
    }
}