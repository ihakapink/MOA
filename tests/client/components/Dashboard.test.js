import React from 'react'
import {shallow, render} from 'enzyme'
import Dashboard from '../../../client/components/Dashboard'


test('<Dashboard/>' , () => {
 const wrapper = shallow(<Dashboard />)
 expect(wrapper.find('h1').text()).toBe('<Header />')
})