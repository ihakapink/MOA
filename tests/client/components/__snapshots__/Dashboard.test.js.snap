// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Dashboard should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
